test/test_cli.py

```diff
@@ -233,10 +233,32 @@ def test_runs(self, caplog):
             main(["cve-bin-tool", test_path, "-r", ",".join(runs)])
         self.check_checkers_log(caplog, skip_checkers, runs)
 
+    @pytest.mark.skipif(not LONG_TESTS(), reason="Update flag tests are long tests")
+    def test_initialize(self, caplog):
+        with caplog.at_level(logging.INFO):
+            main(["cve-bin-tool", "--initialize"])
+        assert (
+            "cve_bin_tool",
+            logging.INFO,
+            "Initializing Database",
+        ) in caplog.record_tuples
+        caplog.clear()
+
     @pytest.mark.skipif(not LONG_TESTS(), reason="Update flag tests are long tests")
     def test_update(self, caplog):
         test_path = str(Path(__file__).parent.resolve() / "csv")
 
+        with caplog.at_level(logging.DEBUG):
+            main(
+                ["cve-bin-tool", "-l", "debug", "-u", "latest", "-n", "json", test_path]
+            )
+        assert (
+            "cve_bin_tool.CVEDB",
+            logging.DEBUG,
+            "Updating CVE data. This will take a few minutes.",
+        ) in caplog.record_tuples
+        caplog.clear()
```

Comment:
I can't figure out why this part of the test was moved up.  Was something not working?

Reply:
After initialize database is used database becomes empty. So the test case is moved up to add data to the database.

Reply:
@Rexbeast2 Understood but maybe having an empty database should be checked?  Each of the test cases should ideally be standalone but it sounds like that is no longer the case.

Reply:
@anthonyharrison ok. I will add a check for empty database after assert statement and add data to database rather than waiting to add new database in next step.

Reply:
Might make more sense to copy & restore the db in this test (and maybe the others as well?) to avoid the situation where the update actually has to re-download the whole thing again.  We should double-check that this is in the symmetric tests and not run asymmetrically, but i think all the test_cli.py ones are so it should be ok.